deployments/icesat2/config/prod.yaml

```diff
@@ -4,6 +4,29 @@ daskhub:
     hub:
       extraEnv:
         OAUTH_CALLBACK_URL: "https://aws-uswest2.pangeo.io/hub/oauth_callback"
+    singleuser:
+      profileList:
+        - display_name: "Latest Pangeo-notebook"
+          description: "https://github.com/pangeo-data/pangeo-docker-images/releases"
+          kubespawner_override:
+            image: pangeo/pangeo-notebook:latest
+        - display_name: "Latest Base-notebook"
+          description: "https://github.com/pangeo-data/pangeo-docker-images/releases"
+          kubespawner_override:
+            image: pangeo/base-notebook:latest
+        - display_name: "Latest ML-notebook"
+          description: "https://github.com/pangeo-data/pangeo-docker-images/releases"
+          kubespawner_override:
+            image: pangeo/ml-notebook:latest
+            mem_limit: 60G
+            mem_guarantee: 25G
+            environment: {'NVIDIA_DRIVER_CAPABILITIES': 'compute,utility'}
+            tolerations: [{'key': 'nvidia.com/gpu','operator': 'Equal','value': 'present','effect': 'NoSchedule'}]
+            extra_resource_limits: {"nvidia.com/gpu": "1"}
```

Comment:
I think applying this resource request will make the toleration automatically be applied by some controller in k8s, but it wont hurt also manually applying the toleration.

Reply:
These are from a merge conflict, but @scottyhq might want to take a look at the comment :)

Reply:
if I understand correctly specifying `extra_resource_limits: {"nvidia.com/gpu": "1"}` automatically sets `tolerations: [{'key': 'nvidia.com/gpu','operator': 'Equal','value': 'present','effect': 'NoSchedule'}]` ?

Reply:
Honestly these settings we're copied over from GCP and we never did two much experimentation to see what was necessary or not. Also things may have changed with more recent AMI versions and CUDA setups. This issue has some additional details https://github.com/jupyterhub/zero-to-jupyterhub-k8s/issues/994#issuecomment-678510805